collections.go

```diff
@@ -0,0 +1,73 @@
+package shopify
+
+import "time"
+
+// Collections is a list of collections
+type Collections []Collection
+
+// Collection is a grouping of products.
+type Collection struct {
+	// ID is the ID for the collection.
+	ID int64
+	// Title is the name of the collection. (limit: 255 characters)
+	Title string
+	// BodyHTML is a description of the collection, complete with HTML markup. Many templates display this on their collection pages.
+	BodyHTML string
+	// Handle is a unique, human-readable string for the collection automatically generated from its title.
+	/* This is used in themes by the Liquid templating language to refer to the collection. (limit: 255 characters) /*
+	Handle string
+	// Image is the image assoicated with collection
+	/*
+		Valid values are:
+		- attachment: An image attached to a collection returned as Base64-encoded binary data.
+		- src: The source URL that specifies the location of the image.
+		- alt: The alternative text that describes the collection image.
+		- created_at: The time and date (ISO 8601 format) when the image was added to the collection.
+		- width: The width of the image in pixels.
+		- height: The height of the image in pixels.
+	*/
```

Comment:
we can remove this comment and put the words on the properties of the new Image type